README.md

```diff
@@ -18,6 +18,9 @@ Run
 
 `java -classpath lib/mxgraph-core.jar:classes com.mxgraph.svg2xml.Svg2XmlGui`
 
+On Windows 
+`java -cp ./classes;. com.mxgraph.svg2xml.Svg2XmlGui`
```

Comment:
I suggest to add the jar in the lib folder, adding the root folder requires to manually move the jar there
```suggestion
`java -cp classes;lib/mxgraph-core.jar com.mxgraph.svg2xml.Svg2XmlGui`
```